Traceback (most recent call last):
  File "/home/op/mesh_deploy_5.80/clear_lane.py", line 36, in <module>
    ip = sys.argv[1]
IndexError: list index out of range
[Finish] Failed to deploy [405]